modules/sql-engine/src/integrationTest/java/org/apache/ignite/internal/sql/engine/ItTemporalIndexTest.java

```diff
@@ -252,7 +252,7 @@ public void testSearchGtLtWithPkIdxUsage(String table, String predicate, Object
         for (String idx : SORTED_INDEXES.get(table)) {
             assertQuery(format("SELECT /*+ FORCE_INDEX({}) */ val FROM {} WHERE pk {} ORDER BY val",
                     idx, table, predicate))
-                    .matches(containsIndexScan("PUBLIC", table, idx))
+                    .matches(containsIndexScanIgnoreBounds("PUBLIC", table, idx))
```

Comment:
For me it looks like we need to improve a bit such tests 😔 
We must ensure that for pk-index search bounds are present.
Possible solutions:
1. add `if (idx.equals(pkIndexName(table)))` and call containsIndexScan
2. (preferable) remove pk-indexes from SORTED_INDEXES indexes collection and add separate pk check with containsIndexScan in each test.